### PR TITLE
Add modifications for a better README

### PR DESCRIPTION
**Pull request by rememberYou**
Thanks for `emacs-slack`, I allowed myself to add some aesthetic and content details for the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
+<p align="center"><b>Emacs Slack</b></p>
+<p align="center">GNU Emacs client for <a href="https://slack.com/">Slack</a>.</p>
 
-Emacs client for [Slack](https://slack.com/)
-### **[ScreenShots](https://github.com/yuya373/emacs-slack/wiki/ScreenShots)**
+---
+
+## Preview
+
+You can see some gif on the [wiki](https://github.com/yuya373/emacs-slack/wiki/ScreenShots).
```
Comment (matthew-piziak) [inline README.md]: Spelling: gif -> gifs

File: README.md [hunk 2]
```diff
@@ -1,19 +1,26 @@
-# emacs-slack
+<p align="center"><img src="assets/slack-logo.svg" width=300 height=126/></p>
```
Comment (matthew-piziak) [inline README.md]: Is this image expected to be broken in GitHub preview?

File: README.md (hunk 1 above)
Comment (rememberYou) [reply README.md]: Oops, I will change that.

File: README.md (hunk 2 above)
Comment (rememberYou) [reply README.md]: I've never used GitHub preview to see an image preview, but I can see it in README. I can change my image if necessary.

Comment (matthew-piziak) [reply README.md]: Cool!

Comment (yuya373) [reply README.md]: Thanks!
I want to separate media files from source code. 
Could you create pull request to [assets branch](https://github.com/yuya373/emacs-slack/tree/assets) including svg in assets folder, and then remove svg from this pull request?

Comment (rememberYou) [reply README.md]: Of course, I'll do that to you right now.

Comment (yuya373) [reply README.md]: img's src attribute will be `https://github.com/yuya373/emacs-slack/raw/assets/assets/slack-logo.svg`

Comment (rememberYou) [reply README.md]: You're right. Excuse me for this awkwardness.

Comment (yuya373) [reply README.md]: need `?sanitize=true`  to display svgs 😢 
`https://raw.githubusercontent.com/yuya373/emacs-slack/assets/assets/slack-logo.svg?sanitize=true`

https://github.com/github/markup/issues/556#issuecomment-399336652

Comment (rememberYou) [reply README.md]: This should be settled now. Sorry about the rehearsal.

Comment (yuya373) [reply README.md]: 🎉

**Pull request merged**